backend/shared/src/websockets/server.ts

```diff
@@ -108,6 +109,10 @@ export function broadcastMulti(topics: string[], data: BroadcastPayload) {
   }
 }
 
+export function broadcastCommentUpdate(commentId: string, data: CommentUpdatePayload) {
+  return broadcast(`comment:${commentId}`, data)
+}
+
```

Comment:
This function should instead be in the `helpers.ts` file within shared/src/websockets